packages/client/test/sync/sync.spec.ts

```diff
@@ -34,6 +35,16 @@ tape('[Synchronizer]', async (t) => {
       t.notOk((sync as any).running, 'stopped')
       t.end()
     })
+    sync.syncTargetHeight = new BN(1)
+    setTimeout(() => {
+      // eslint-disable-next-line no-extra-semi
+      ;(sync as any).chain._blocks = {
+        latest: { hash: () => Buffer.from([]) },
+        td: new BN(0),
+        height: new BN(1),
+      }
+      config.events.emit(Event.CHAIN_UPDATED)
+    }, 100)
```

Comment:
Ah, pretty cool fix.

Reply:
yeah a little hacky but was the best way I could find to easily override for now